pkg/apis/admissionregistration/validation/validation.go

```diff
@@ -231,17 +231,20 @@ func ValidateMutatingWebhookConfiguration(e *admissionregistration.MutatingWebho
 		requireNoSideEffects:                    true,
 		requireRecognizedAdmissionReviewVersion: true,
 		requireUniqueWebhookNames:               true,
+		allowInvalidLabelValueInSelector:        true,
```

Comment:
we *don't* want to allow invalid values on creation

Reply:
set allowInvalidLabelValueInSelector to false in ValidateValidatingWebhookConfiguration as well